settings.gradle

```diff
@@ -1,4 +1,5 @@
 include ':examples',
+        ':app-tests-wrapper',
```

Comment:
suggest to move this line to bottom. Keep test modules together at this list

Reply:
it's `application` module. Maybe keep it with other  apps?